development-standards.md

```diff
@@ -0,0 +1,446 @@
+
+## 🧩 Components
+
+Components live in the `components` package of our project. Each component should be a directory and contain the following files and folders:
+
+- `index.tsx` - The compoent logic and main ui
+- `index.spec.txt` - The tests for the component
+- `index.stories.tsx` - The Storybook file for the component
+- `styles.module.scss` - The scoped component styling
+- `/images` - Any component specific images
+- `/helpers` - Additional logic for the component which makes sense to be separated from the index (ensure tests also exist for these)
+
+Any other component specific assets may also be placed in this directory.
+
+The only required files are index.tsx, index.spec.tsx and index.story.tsx. Please ensure you stick to the above naming conventions.
+
+Each component should have a single task (See Brad Frost's [Atomic Design Principle](https://bradfrost.com/blog/post/atomic-web-design/) for more information) and be written as a 'pure' component. This means that it should only use react's hooks and context api along with props to receive and pass information. (See React Coding standards below)
+
+> ⚠️ Please note: Components should not contain other components, they should only reference them. If you are making a component, it should be created in the root of `/components` and have a Storybook entry.
+
+## 📖 Storybook
+
+<div align="center">
+
+[![Storybook intro video on YouTube](https://i.imgur.com/FDvR6zl.jpg)](https://www.youtube.com/watch?v=LFh5Y89eM)
+
+</div>
+
+All components, no matter how small or large should have a Storybook entry. Ideally a component should be built in insolation and tested within Storybook before being imported into the primary project (or that components parent component), this ensures that the component is robost and works by itself. It also provides a handy component library for developers to be able to reuse components and prevent the reinvention of the wheel.
+
+Rather than document how to use Storybook, please refer to the excellent [Storybook Documentation](https://storybook.js.org/docs/basics/introduction/)
+
+## 👩‍💻 Coding best practice
+
+### General coding
+
+This section applies to the entire project, regardless of language
+
+#### Coding style
+
+- Use spaces for tabs
+- Use 2 space widths per tab indentation
+- Trim all trailing whitespace
+
+> ❗ For the simplest development process, please ensure youre development environment confirms to the project `.editorconfig` file. There are plugins for most (if not all popular editors), find yours [here](https://editorconfig.org/#download).
+
+#### Code comments
+
+Code comments should be written in a formal, imperative style and describe the code or the intended outcome. Please avoid the use of colloquialisms and use present tense.
```

Comment:
Could you give a couple of examples of good/bad use of `formal, imperative style`?  I'll have to work hard to adopt it, our non-native English speakers may find it even more challenging.